portfolio/src/main/webapp/index.html

```diff
@@ -98,6 +97,38 @@ <h2>A business-focused technologist.</h2>
     <div class="section-about u-padding-sm u-text-shadow">
       <h1>In a rush?</h1>
       <h2>Learn about me in 30 seconds or less.</h2>
-      <p> Lorem Ipsum Emmet.
+      <p> Lorem Ipsum Emmet.<br>
       <div id="response-container"></div>
     </div>
+    <div>
+      <form action="/data" method="POST">
+        <p>Introduce yourself!</p>
```

Comment:
Nit: `</p>`

Reply:
</p> must be present since it is followed by a <label> element

https://html.spec.whatwg.org/multipage/syntax.html#void-elements

Reply:
Wasn't aware of that. Thanks for looking that up!